README.md

```diff
@@ -88,6 +116,44 @@ To deploy PromDash behind a reverse proxy you can set a global path prefix
 using the environment variable `PROMDASH_PATH_PREFIX`. Once set all URLs will
 start with the given prefix.
 
+One simple way to secure your Prometheus deployment with authentication is
+running a reverse proxy, Prometheus, and PromDash on the same machine. The goal
+is to have a publicly available deployment for authenticated users via your
+reverse proxy while restricting raw `IP:PORT` access. We won't go cover any
+details on how to configure your reverse proxy, as it depends on what you are
+using.
+
+**Reverse proxy** (e.g. [nginx](http://nginx.org/), [HAProxy](http://www.haproxy.org/))
+* Set up two domains. Route requests by the specific hostnames and reject any other requests.
+ * `http(s)://prometheus.example.com` to backend `localhost:9090`
+ * `http(s)://dash.example.com` to backend `localhost:3000`
+* Configure ACL, e.g. HTTP basic auth for both backends
+
+**Run Prometheus** with binding only to localhost to restrict direct external
+access over the network that would circumvent your authentication. Remember to
+pass `-web.external-url`. Example for Docker:
+([full reference](http://prometheus.io/docs/introduction/install/#using-docker))
+
+    docker run -p 127.0.0.1:9090:9090 -v /prometheus-data \
+           prom/prometheus -config.file=/prometheus-data/prometheus.yml \
+                           -web.external-url "http://prometheus.example.com/"
+
+**Running PromDash** with Docker's default `--net brigde` won't let it poll
+Prometheus at `localhost:9090` from the host. We will need to use `--net host`
```

Comment:
I didn't notice this earlier. As mentioned in https://github.com/prometheus/promdash/issues/341, PromDash does *not* need to talk to Prometheus at all. It only needs to be able to talk to its database (and perhaps an annotation server, if that's optionally configured, but let's ignore that here). Is the `--net host` setting still needed here then for other reasons?